src/components/DetailModal/index.js

```diff
@@ -15,6 +15,9 @@ const DetailModal = ({data, setData, skills, setSkills, selectedID}) => {
                                     aria-label="Close"></button>
                         </div>
                         <div className="modal-body">
+                            {loading && <p>Loading...</p>}
+                            {error && <p>Error: {error.message}</p>}
+                            {!loading && !error && (
                             <div className="container">
                                 <div
                                         className="d-flex flex-sm-column flex-lg-row align-items-start bg-primary text-white py-5 fs-4">
```

Comment:
Consistency: Why are the attributes on a separate line and indented so far? And this looks like its thrown the end tag out so its not lined up with the opening tag. Which has a knock on effect to following tags.